lib/api/index.js

```diff
@@ -5,3 +5,6 @@ module.exports.stream = require('./api-stream')
 module.exports.pipeline = require('./api-pipeline')
 module.exports.upgrade = require('./api-upgrade')
 module.exports.connect = require('./api-connect')
+
+module.exports.responseErrorInterceptor = require('../interceptor/response-error')
+module.exports.retryInterceptor = require('../interceptor/retry')
```

Comment:
```suggestion
module.exports.interceptors = { retry: require('../interceptor/retry'), throwOnError: require('../interceptor/response-error') }
```

Reply:
@metcoder95 wdyt?

Reply:
Just to have more context, why are we exposing them through the API instead of just the normal interceptors as we already have? https://github.com/nodejs/undici/blob/9134ee653af1ac933df975d76883fefd92bc23c2/index.js#L41